### DISCUSSION TITLE
release

### DISCUSSION THREAD

**Discussion by marco-santulli** [Budibase/budibase#5676] [General]
Hi,

is there any plan to do a new build for self-hosters?

Ciao

M

**Comment by mike12345567** [comment]
Hi @marco-santulli, see my answer here: https://github.com/Budibase/budibase/discussions/5653

We will be performing a release soon, we have just made a lot of core changes with this release and we are still testing it.

**Reply by marco-santulli** [reply]
hi mike,

no problem I was just wondering if it was in planning to be able to plan un update on my side

thx

Marco
